--- a/jdk/make/lib/Awt2dLibraries.gmk	2017-07-27 13:04:48.000000000 -0500
+++ b/jdk/make/lib/Awt2dLibraries.gmk	2017-09-12 22:10:29.881578950 -0500
@@ -249,7 +249,7 @@
         -delayload:ole32.dll -delayload:comdlg32.dll \
         -delayload:comctl32.dll -delayload:shlwapi.dll, \
     LIBS_unix := -ljvm -ljava $(LIBM), \
-    LIBS_linux :=  $(LIBDL), \
+    LIBS_linux :=  $(LIBDL), -lfontconfig, \
     LIBS_solaris := $(LIBDL) -lc, \
     LIBS_aix := $(LIBDL),\
     LIBS_macosx := -lmlib_image \
